src/storage/mvcc/reader/scanner/txn_entry.rs

```diff
@@ -304,7 +307,7 @@ impl<S: Snapshot> Scanner<S> {
             let value = super::super::util::near_load_data_by_write(
                 default_cursor,
                 user_key,
-                write,
+                write.start_ts,
```

Comment:
It seems that we do not need to pass `Write` to `load_data_and_write `. Because there is nothing with copy of `short_value`. Why not `WriteRef`

Reply:
Because WriteRef keeps a reference which will break the borrow checker.